package.json

```diff
@@ -31,12 +32,13 @@
     "@types/express-session": "^1.18.0",
     "@types/mocha": "^10.0.7",
     "@types/mongoose": "^5.11.96",
-    "@types/node": "^20.16.1",
+    "@types/node": "^20.16.5",
     "@types/nunjucks": "^3.2.6",
     "@types/pbkdf2": "^3.1.2",
+    "@types/selenium-webdriver": "^4.1.26",
     "@types/sinon": "^17.0.3",
     "axios-mock-adapter": "^1.21.4",
-    "chai": "^4.3.7",
+    "chai": "^4.5.0",
```

Comment:
Making this change will crash the authorisation, and a few other features

Reply:
will change if crashes